types/ajv-merge-patch/ajv-merge-patch-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as Ajv from "ajv";
+import * as ajvMergePatch from "ajv-merge-patch";
```

Comment:
Should use `import ajvMergePatch = require("ajv-merge-patch");` here

Reply:
May I ask why should remove `declare namespace ajvMergePatch {}` and use `import ajvMergePatch = require("ajv-merge-patch");`?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
Thanks! :stuck_out_tongue_closed_eyes: